module/lib/roll.js

```diff
@@ -181,7 +183,6 @@ export function prepareRollDialog(options) {
 
   options.penalty = 0;
   options.bonusDefault = 0;
-  options.dicePool = 0;
```

Comment:
I think this was a mistake introduced in the explosive work. The rolls setting the dicePool size in the sheet before calling the `prepareRollDialog` method were not working.